lib/dialogs/missions/missionsChoiceStatus.js

```diff
@@ -0,0 +1,57 @@
+/* global module */
+var builder = require("botbuilder");
+
+module.exports = function (bot) {
+    bot.dialog("missionsStatus", require("./missionsStatus"));
+    var choices = {
+        "Accepted" : {
+            item: "Accepted"
+        },
+        "Declined": {
+            item: "Declined"
+        },
+        "Pending": {
+            item: "Pending"
+        },
+        "All": {
+            item: "All"
+        }
+    };
+
+    var statusText = [
+        "Declined","Accepted","Pending"
+    ]
+ 
+    return [
+        function (session, args) {
+            if (args && args.reprompt) {
+                builder.Prompts.text(session, `I'm affraid I don't have anything to show you. Shall we look another status ?`);
+            }
+            else {
+                builder.Prompts.choice(session, `Ok ${session.userData.data.firstName}, which status of missions do you want to retrieve ?`, choices, {listStyle: builder.ListStyle["button"]});
```

Comment:
on a une fonction qui récupère automatiquement le nom sans risque de péter dans utils/session

Reply:
session.send(`Well hello to you ${getUserName(session)}`);